features/list_companies_of_category.feature

```diff
@@ -0,0 +1,36 @@
+Feature: As a visitor
```

Comment:
You must be explicit about whether someone is logged in or not for a scenario.  Always have either 'I am logged out' or 'I am logged in as "...."'

Please add scenarios so we have all of theses possibilities covered:

- A visitor views a category
- A user views a category
- A member views a category
- An admin views a category

- A visitor tries to view a category with no companies
- A visitor views a category with 3 companies (You should verify that they see _3_ companies)



Remember that this _feature_ should describe what happens in all cases, for all types of users.   We need a description of what happens in all possibilities so that:
- we know we've thought through the feature completely,
- others reading this in the future and trying  to understand what should happen with this feature get a complete picture,
**and importantly:**
- to be _sure_ that in the future as the system grows in complexity, we are testing everything.